src/config/nav.tsx

```diff
@@ -19,6 +19,11 @@ export const viewerNavItems = [
     name: 'New',
     to: absoluteRoutes.viewer.new(),
   },
+  {
+    icon: <SvgNavDiscover />,
+    name: 'Discover',
+    to: absoluteRoutes.viewer.discover(),
+  },
```

Comment:
Let's hide this for now so we can deploy `dev` if needed before the featured content is ready